rustzx-core/src/host/mod.rs

```diff
@@ -23,7 +23,7 @@ pub enum SnapshotRecorder<DataRecorderImpl: DataRecorder> {
 
 pub enum Tape<LoadableAssetImpl: LoadableAsset> {
     Tap(LoadableAssetImpl),
-    // TODO(#56): Implement TZX tape format support
+    Tzx(LoadableAssetImpl), // TODO(#56): Implement TZX tape format support
```

Comment:
Remove TODO

Reply:
Also, please add "closes #56" line to PR description so this issue will close automatically